cmd/svcat/svcat_test.go

```diff
@@ -92,16 +92,24 @@ func TestCommandOutput(t *testing.T) {
 		continueOnError bool   // Should the test stop immediately if the command fails or continue and capture the console output
 	}{
 		{name: "list all brokers", cmd: "get brokers", golden: "output/get-brokers.txt"},
+		{name: "list all brokers (json)", cmd: "get brokers -o json", golden: "output/get-brokers.json"},
+		{name: "list all brokers (yaml)", cmd: "get brokers -o yaml", golden: "output/get-brokers.yaml"},
 		{name: "get broker", cmd: "get broker ups-broker", golden: "output/get-broker.txt"},
+		{name: "get broker (json)", cmd: "get broker ups-broker -o json", golden: "output/get-broker.json"},
+		{name: "get broker (yaml)", cmd: "get broker ups-broker -o yaml", golden: "output/get-broker.yaml"},
 		{name: "describe broker", cmd: "describe broker ups-broker", golden: "output/describe-broker.txt"},
 
 		{name: "list all classes", cmd: "get classes", golden: "output/get-classes.txt"},
+		{name: "list all classes (json)", cmd: "get classes -o json", golden: "output/get-classes.json"},
+		{name: "list all classes (yaml)", cmd: "get classes -o yaml", golden: "output/get-classes.yaml"},
 		{name: "get class by name", cmd: "get class user-provided-service", golden: "output/get-class.txt"},
```

Comment:
There is a missing testcase for outputting yaml/json for "get class by name". No need to do it for uuid too.